server/src/graphql/mutations/loginWithGithub.ts

```diff
@@ -80,6 +77,7 @@ export const loginWithGithub: MutationResolvers['loginWithGithub'] = async (
         avatarUrl: githubUser.avatar_url,
         email: email.email,
         githubAccessToken: data.access_token,
+        // TODO save refresh token + refresh_token_expires_in?
```

Comment:
@pradel should we create separate issue for this?

Reply:
@Akirtovskis yeah good idea to not forget about it 👍